sql/_13_issues/_25_1h/cases/cbrd_25909.sql

```diff
@@ -0,0 +1,407 @@
+
+
+call login('dba','') on class db_user;
+ALTER VIEW u2.v1_tbl OWNER TO u3;
+select * from db_auth where grantee_name != 'PUBLIC' order by 1;
+select owner.name, grants from db_authorization where owner.name != 'PUBLIC' order by 1;
+select * from db_class where class_name ='v1_tbl';
+show create view u3.v1_tbl;
+
+call login(class db_user,'dba','');
+drop table u1.tbl;
+drop view u3.v1_tbl;
+
+
+
+evaluate '3. change to stored procedure owner';
+create function u1.hello() return string
+as language java
+name 'SpCubrid.HelloCubrid() return java.lang.String';
+
+call login('u1','') on class db_user;
+GRANT EXECUTE ON PROCEDURE u1.hello TO u2;
+GRANT EXECUTE ON PROCEDURE u1.hello TO u3;
+
+select * from db_auth where grantee_name != 'PUBLIC' order by 1;
+
+select owner.name, grants from db_authorization where owner.name != 'PUBLIC' order by 1;
+
+call login('u3','') on class db_user;
+evaluate 'check error message';
+ALTER FUNCTION u1.hello OWNER TO u3;
+ALTER FUNCTION u1.hello OWNER TO u3 COMMENT 'aaaa';
+ALTER FUNCTION u1.hello COMMENT 'aaaa';
+DROP FUNCTION u1.hello;
+
+
+call login('u1','') on class db_user;
+evaluate 'success';
+ALTER FUNCTION u1.hello COMMENT 'aaaa';
+select sp_name, owner, comment from db_stored_procedure;
+
+
+call login('dba','') on class db_user;
+ALTER FUNCTION u1.hello OWNER TO u3;
+
+select * from db_auth where grantee_name != 'PUBLIC' order by 1;
+select owner.name, grants from db_authorization where owner.name != 'PUBLIC' order by 1;
+select sp_name, owner from db_stored_procedure;
+
+call login(class db_user,'dba','');
+drop FUNCTION u3.hello;
+
+
+
+evaluate '4. Change the owner of a table to which you have granted ALTER permission';
+create table u1.tbl (col1 int);
+create table u1.tbl2 (col1 int);
+
+
+call login('u1','') on class db_user;
+GRANT ALTER ON u1.tbl TO u2;
+GRANT ALTER ON u1.tbl2 TO u2 WITH GRANT OPTION;
+
+
+call login('u2','') on class db_user;
+GRANT ALTER ON u1.tbl2 TO u3 WITH GRANT OPTION;
+
+
+call login('dba','') on class db_user;
+select * from db_auth where grantee_name != 'PUBLIC' order by 1;
+select owner.name, grants from db_authorization where owner.name != 'PUBLIC' order by 1;
+
+
+call login('u3','') on class db_user;
+evaluate 'check error meessage';
+ALTER TABLE u1.tbl OWNER TO u3;
+ALTER TABLE u1.tbl2 OWNER TO u3;
+
+
+call login('u2','') on class db_user;
+evaluate 'check error meessage';
+ALTER TABLE u1.tbl OWNER TO u3;
+ALTER TABLE u1.tbl2 OWNER TO u3;
+
+
+call login('dba','') on class db_user;
+ALTER TABLE u1.tbl OWNER TO u3;
+ALTER TABLE u1.tbl2 OWNER TO u3;
+
+select * from db_auth where grantee_name != 'PUBLIC' order by 1;
+select owner.name, grants from db_authorization where owner.name != 'PUBLIC' order by 1;
+select class_name, owner_name from db_class where class_name in ('tbl', 'tbl2') order by 1;
+
+call login(class db_user,'dba','');
+drop table u3.tbl;
+drop table u3.tbl2;
+
+
+
+evaluate '5. change to the owner for partition table'
```

Comment:
```suggestion
evaluate '5. change to the owner for partition table';
```
이로인해 하위 테스트가 모두 의도와 다르게 테스트 되고 있습니다.